src/DynamoCore/Interfaces/IPathResolver.cs

```diff
@@ -98,6 +98,11 @@ public interface IPathManager
         string SamplesDirectory { get; }
 
         /// <summary>
+        /// The directory where the automatically saved files will be stored.
+        /// </summary>
+        string BackupDirectory { get; }
+
+        /// <summary>
```

Comment:
`IPathResolver` is being implemented by classes in `DynamoSandbox.exe`, `DynamoStudio.exe`, Revit add-ons, Revit tests. Please be sure those are updated immediately after this one is merged.